packages/core/state/selection/logics.ts

```diff
@@ -418,6 +439,18 @@ const selectNearbyFile = createLogic({
     type: [SELECT_NEARBY_FILE],
 });
 
+/**
+ * Interceptor responsible for processing the changed collection into
+ * a refresh action so that the resources pertain to the current collection
+ */
+const changeCollectionLogic = createLogic({
+    async process(_: ReduxLogicDeps, dispatch, done) {
+        dispatch(interaction.actions.refresh() as AnyAction);
```

Comment:
Having a logic that only dispatches another action seems like a smell to me. Could this be handled instead by adding some logic to the reducer, or by having a different logic handle both the refresh action and this one?